VCS/1. User Guides/15. 管理資料磁碟.md

```diff
@@ -1 +1,212 @@
+---
+tags: Guide, VCS, VDS, EN
+title: 管理磁碟 | en
+GA: UA-155999456-1 
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# Manage data disks
+
+After the **data disk** is created, you can view the disk information, attach the disk to (or detach from) the VCS instance, and delete the disk. <br>(*※ System disk currently does not support the following management functions*.)
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+For the difference between Tenant Admin and Tenant User for using VCS instance, see [<ins>User roles and permissions</ins>](https://man.twcc.ai/@twccdocs/role-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Frole-storage-en#%E8%99%9B%E6%93%AC%E7%A3%81%E7%A2%9F%E6%9C%8D%E5%8B%99).
+:::
+
+## View the data disk
+
+View the ID, created time, name, size, type, state,  description, attached VCS instance, and mount path of your data disk.
+
+<!-- 1 start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC Portal</b></summary>
+
+<br>
+
+- Enter **Data Disk Management** page, then you can view the summary information of the diks. 
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_bd8f238dd56b9bd437f4b3c2dbbbccbf.png)
+
+
+- Click on one of your data disks and enter the **Data Disk Details** page, then you can view the detailed disk information.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_a56cfee1ff28f578bff894daed2d5a62.png)
+
+
+
+</details>
+
+<!-- Space -->
+
+<div style="height:8px"></div>
+
+<!-- 2. start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC CLI</b></summary>
+
+<br>
+
+- View the ID, name, created time, mount path, size, state and type of the disk.
+
+```bash
+$ twccli ls vds
+```
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_c976443120fa105196269359143aeb3a.png)
+
+</details>
+
+## Attach to VCS instance
+
+After the data disk is created, you can attach the disk to the VCS instance then initialize the disk to use.
+
+<!-- 1 start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC Portal</b></summary>
+
+<br>
+
+*  After the data disk is created, it will appear at the top of the Data Disk Management list. When its state becomes **`AVAILABLE`**, you can attach to the instance for use. 
```

Comment:
[SUG]

After the data disk is created, it will appear at the top of the Data Disk Management list. >

The newly created data disk will appear at the top of the Data Disk Management list.

Reply:
[SUG]

1. When its state becomes **`AVAILABLE`** > When it enters the **`AVAILABLE`** state
2. you can attach to the instance for use. > you can attach it to an instance for use.